### DISCUSSION TITLE
Disabling some Javascript in internal side-by-side preview

### DISCUSSION THREAD

**Discussion by pmioni** [sulu/sulu#6453] [Q&A] [answered]
Q  
--
| Bug? | no
| New Feature? | no
| Sulu Version 2.3.9 

Hi, we would like to disable some animations in the front-end of our project when it is shown in Sulu's side by side preview in the administration interface. How can we do that? Is there a way we can recognize in which environment we're running? Is there some variable passed to the views for that use case?

**Comment by alexander-schranz** [comment, marked as answer]
@pmioni Into the controller a `$preview` variable is injected to know if you are in preview or not: 
https://github.com/sulu/sulu/blob/ee6e61de48f20d1514e4f2bf9fc49401a5882000/src/Sulu/Bundle/WebsiteBundle/Controller/DefaultController.php#L31

In twig you can access this variable by using:

```twig
{% if not app.request.attributes.get('preview', false) %}
    <!-- website only -->
{% endif %}
```

**Comment by pmioni** [comment]
Thanks @alexander-schranz it works perfectly!
